clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -202,6 +202,10 @@ Changes in existing checks
   <clang-tidy/checks/bugprone/reserved-identifier>` check, so that it does not
   warn on macros starting with underscore and lowercase letter.
 
+- Improved :doc:`bugprone-unchecked-optional-access
+  <clang-tidy/checks/bugprone/unchecked-optional-access>` check, use `cast_or_null`
+  instead `cast` so that it does not crash during handling of optional values.
```

Comment:
information about cast or cast_or_null is not needed here, this is just info for users who had this check crashed, and decided to disable it, to consider re-enabling it or even testing it again on their projects.